.travis.yml

```diff
@@ -39,19 +39,20 @@ jobs:
         - ./.travis_test.sh
       env:
         - RUN_INTEGRATION_TESTS=false
-    - name: "QuickStart - OracleJDK 8"
+    - name: "QuickStart - Java 8 & OpenJDK 14-15"
```

Comment:
Why splitting the tasks this way? How about the other OracleJDK versions?

Reply:
Just don't want to have many parallel Quickstart jobs.
Also tries to keep the test time of one job to not exceed unit test/integration test.